content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_SQL_Server_Deployment_Guide--Enabling_Always_Encrypted_with_YubiHSM_2/Getting_Help_and_Further_Reading.adoc

```diff
@@ -0,0 +1,28 @@
+== Getting Help and Further Reading
+
+Should you require assistance when using this guide to deploy YubiHSM 2 with Microsoft SQL Server Always Encrypted, start by referencing the product documentation and currently known issues:
+
+link:../../developers.yubico.com/[Yubico Developers website]
+
+link:https://support.yubico.com/support/home[Yubico Support]
+
+link:../../developers.yubico.com/YubiHSM2/[Product documentation]
+
+link:../..developers.yubico.com/YubiHSM2/Releases/Known_issues.adoc[Known issues and limitations]
```

Comment:
this link is broken, it should be ../../Releases/Known_issues.adoc

Reply:
indeed. Fixed.